editor/i18n/en/assets.js

```diff
@@ -318,10 +318,16 @@ module.exports = {
             preferLocalTimeSpan: {
                 name: 'Prefer Local Time Span',
                 title:
-                    'When exporting FBX animations, whether prefer to use the time range recorded in FBX file. ' +
-                    'If one is not preferred, or one is invalid for use, the time range is robustly calculated. ' +
+                    'When exporting FBX animations, whether prefer to use the time range recorded in FBX file.<br>' +
+                    'If one is not preferred, or one is invalid for use, the time range is robustly calculated.<br>' +
                     'Some FBX generators may not export this information.',
             },
+            smartMaterialEnabled: {
+                name: 'Smart Material Conversion',
+                title: 'Convert DCC material to engine material, and match the lighting model of DCC material.',
+                warn: 'For the Smart Material Conversion of laboratory function in preferences is turned off, ' +
```

Comment:
Should forbid changing this option when laboratory feature is disabled

Reply:
warning text only showing when laboratory feature is disabled:

"The Smart Material Conversion feature have been disabled in laboratory preference, please enable it to change model level setting."